test/LibraryManager.IntegrationTest/LibmanCompletionTests.cs

```diff
@@ -33,5 +32,152 @@ public void LibmanCompletion_Destination()
 
             LibmanTestsUtility.WaitForCompletionEntries(Editor, expectedCompletionEntries, caseInsensitive: true);
         }
+
+        [TestMethod]
+        public void LibmanCompletion_Provider()
+        {
+            _libManConfig.Open();
+
+            string[] expectedCompletionEntries = new[] {
+                "cdnjs",
+                "filesystem",
+                "unpkg",
+            };
+
+            Editor.Caret.MoveToBeginningOfFile();
+            Editor.Caret.MoveDown(4);
+            Editor.KeyboardCommands.Type("\"provider\":");
+
+            LibmanTestsUtility.WaitForCompletionEntries(Editor, expectedCompletionEntries, caseInsensitive: true);
+        }
+
+        [TestMethod]
+        public void LibmanCompletion_DefaultProvider()
+        {
+            _libManConfig.Open();
+
+            string[] expectedCompletionEntries = new[] {
+                "cdnjs",
+                "filesystem",
+                "unpkg",
+            };
+
+            Editor.Caret.MoveToBeginningOfFile();
+            Editor.Caret.MoveDown();
+            Editor.Caret.MoveToEndOfLine();
+            Editor.KeyboardCommands.Enter();
+            Editor.KeyboardCommands.Type("\"defaultProvider\":");
+
+            LibmanTestsUtility.WaitForCompletionEntries(Editor, expectedCompletionEntries, caseInsensitive: true);
+        }
+
+        [TestMethod]
+        public void LibmanCompletion_DefaultDestination()
+        {
+            _libManConfig.Open();
+            string[] expectedCompletionEntries = new[] {
+                "Properties/",
+                "wwwroot/",
+            };
+
+            Editor.Caret.MoveToBeginningOfFile();
+            Editor.Caret.MoveDown();
+            Editor.Caret.MoveToEndOfLine();
+            Editor.KeyboardCommands.Enter();
+            Editor.KeyboardCommands.Type("\"defaultDestination\":");
+
+            LibmanTestsUtility.WaitForCompletionEntries(Editor, expectedCompletionEntries, caseInsensitive: true);
+        }
+
+        [TestMethod]
+        public void LibmanCompletion_LibraryForCdnjs()
+        {
+            _libManConfig.Open();
+            string[] expectedCompletionEntries = new[] {
+                "bootstrap",
```

Comment:
I believe we're removing this since it's a hacked alias for "twitter-bootstrap".  We can probably just remove it from this test and only rely on jquery.

Reply:
I see from Joe's sample json file contains this library. Let's check it tomorrow.

---
In reply to: [193925241](https://github.com/aspnet/LibraryManager/pull/108#discussion_r193925241) [](ancestors = 193925241)

Reply:
Yes, it is there today, but we have an open issue to remove it soon.  So instead of adding a test scenario that we know will break, let's avoid that confusion when the change comes in.

Reply:
Verified with Joe his bootstrap is used for unpkg. I will remove this case here for cdnjs

---
In reply to: [193932770](https://github.com/aspnet/LibraryManager/pull/108#discussion_r193932770) [](ancestors = 193932770,193925241)

Reply:
Will do

---
In reply to: [194119286](https://github.com/aspnet/LibraryManager/pull/108#discussion_r194119286) [](ancestors = 194119286)